files/bin/gc_builders

```diff
@@ -10,9 +10,9 @@ timestamp() {
 {
   echo "Removing containers"
 
-  for filter in "name=builder-init-*" "name=builder-cloner-*" "name=builder-pull-engines-*" "label=com.codeclimate.role=builder" "name=cc-engines-*"; do
+  for filter in "name=builder-*" "label=com.codeclimate.role=builder" "name=cc-engines-*"; do
     printf "Removing containers with %s\n" "$filter"
-    docker ps -a -f "$filter" | awk '/day|week|month/ { print $1 }' \
+    docker ps -a -f "$filter" --filter "status=exited"| awk '/day|week|month/ { print $1 }' \
```

Comment:
I'm glad we're using the more explicit `--filter`, but can we be consistent and use the long options for all the flags passed here?